app/views/eventRegistrations.html

```diff
@@ -19,7 +19,7 @@ <h2 class="page-title">Registrations</h2>
           </span>
         </div>
       </div>
-      <div class="col-md-2 col-sm-4 stacked-spacing-col-sm">
```

Comment:
I think it would be good to keep one of these `stacked-spacing-*` classes. They add space at a certain viewport size. There could be more space between payment status and registrant type. I'm not seeing any spacing issues between the other inputs so you should only need to keep one.
<img width="518" alt="screen shot 2016-07-21 at 7 29 58 am" src="https://cloud.githubusercontent.com/assets/756501/17021262/f7a55580-4f14-11e6-8de9-41df9fafec7a.png">